test/callback.jl

```diff
@@ -42,6 +42,39 @@ context("With solver $(typeof(lazysolver))") do
     @fact getvalue(y) --> roughly(2.0, 1e-6)
 end; end; end
 
+facts("[callback] Test local lazy constraints") do
+for lazylocalsolver in lazylocal_solvers
+context("With solver $(typeof(lazylocalsolver))") do
+    entered = [false,false]
+
+    weights = [24714888; 21118272; 5063487; 23450813; 5598179; 4049178; 13516450; 8385365; 9684076; 31317634; 14084148; 21750211; 29261668; 17996589; 12115244]
+    values = weights
+    W = floor(sum(weights)/2) # knsapsack instance generated according to criteria in 'Hard knapsack Instances' (Chvatal, Op. Res., 1980), with weights composed of random integers in [1:10^(nbItems / 2)]
+    mod = Model(solver=lazylocalsolver)
+    @variable(mod, 0 <= x[1:length(weights)] <= 1, Int)
+    @objective(mod, Max, dot(x, values))
+    @constraint(mod, dot(x, weights) <=  W)
+
+    global lazy_cutcount_ = 0
```

Comment:
See the approach used with the ``entered`` array. This is better than using global variables. It would be something like ``lazy_cutcount_ = [0]`` and then update ``lazy_cutcount_[1]`` inside the callback.